src/azure-cli/azure/cli/command_modules/hdinsight/custom.py

```diff
@@ -139,6 +142,38 @@ def create_cluster(cmd, client, cluster_name, resource_group_name, cluster_type,
         raise CLIError('Either the kafka client group id and kafka client group name should be specified, '
                        'or none of them should be')
 
+    # Validate and initialize autoscale setting
+    autoscale_configuration = None
+    if autoscale_type == 'Load':
+        if not all([autoscale_min_count, autoscale_max_count]):
+            raise CLIError(
+                'When the autoscale type is Load, both of the autoscal min count and max count should be specified.')
```

Comment:
>min count and max count [](start = 71, length = 23)

I think using --autoscale-min-count  and --autoscale-max-count is better

Reply:
fixed.

---
In reply to: [466794944](https://github.com/Azure/azure-cli/pull/14692#discussion_r466794944) [](ancestors = 466794944)

Reply:
changed it to --autoscale-min-workernode-count and --autoscale-max-workernode-count

---
In reply to: [466795931](https://github.com/Azure/azure-cli/pull/14692#discussion_r466795931) [](ancestors = 466795931,466794944)